tests/test_alloydb_vectorstore_search.py

```diff
@@ -149,6 +149,14 @@ async def test_asimilarity_search_score(self, vs):
         assert results[0][0] == Document(page_content="foo")
         assert results[0][1] == 0
 
+    async def test_similarity_search_with_relevance_scores_threshold(self, vs):
+        score_threshold = {"score_threshold": 0.9}
```

Comment:
Can we add a few tests to show the threshold changing the results?

Reply:
Added tests